charts/generic-service/templates/batch-cronjob.yaml

```diff
@@ -13,6 +13,7 @@ metadata:
 spec:
   suspend:  {{ $job.suspend | default false }}
   schedule: {{ $job.schedule | quote }}
+  timeZone: "Europe/London"
```

Comment:
All of the jobs using this template were running an hour late (BST) but I didn't notice because I'm so used to things being an hour out, e.g. App Insights queries. 😢